<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceSettings</class>
 <widget class="QWidget" name="InterfaceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="layoutParent">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>729</width>
        <height>2402</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelTheme">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Theme</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboTheme">
         <item>
          <property name="text">
           <string>System</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Light</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dark</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSystemIcons">
         <property name="toolTip">
          <string>Uses system icons when available on Linux.</string>
         </property>
         <property name="text">
          <string>Use system icons</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSubSearch">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Subtitle Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameSubSearch">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="layoutSubFont">
            <item>
             <widget class="QLabel" name="labelSubFont">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Font</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFontComboBox" name="fontComboSub">
              <property name="writingSystem">
               <enum>QFontDatabase::WritingSystem::Japanese</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkSubBold">
              <property name="text">
               <string>Bold</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkSubItalic">
              <property name="text">
               <string>Italics</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="layoutSubtitleSearch">
            <item row="1" column="8">
             <widget class="QToolButton" name="buttonSubStroke">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="toolTip">
               <string>The color of the stroke.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="7" alignment="">
             <widget class="QLabel" name="labelSubStrokeColor">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Stroke Color</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QDoubleSpinBox" name="spinSubScale">
              <property name="toolTip">
               <string>How large the text should appear relative to the size of the player.
A value of 0.1 would make each character 10% of the height of the player.</string>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="7" alignment="">
             <widget class="QLabel" name="labelSubStrokeSize">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Stroke Size</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" alignment="">
             <widget class="QLabel" name="labelSubScale">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Text Scale</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4" alignment="">
             <widget class="QLabel" name="labelSubOffset">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Bottom Offset</string>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="4" alignment="">
             <widget class="QLabel" name="labelSubBackground">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Background Color</string>
              </property>
             </widget>
            </item>
            <item row="0" column="9">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1" alignment="">
             <widget class="QLabel" name="labelSubColor">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Text Color</string>
              </property>
             </widget>
            </item>
            <item row="0" column="8">
             <widget class="QDoubleSpinBox" name="spinSubStroke">
              <property name="toolTip">
               <string>Size of the stroke around text.
A value of 0 disables stroke.</string>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="5">
             <widget class="QDoubleSpinBox" name="spinSubOffset">
              <property name="toolTip">
               <string>How high the subtitle search should appear from the bottom relative the size of the player.
A value of 0 positions the subtitle at the bottom.
A value of 1 positions the subtitle at the top.</string>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QToolButton" name="buttonSubColor">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="toolTip">
               <string>The color of the text.</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="5">
             <widget class="QToolButton" name="buttonSubBackground">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="toolTip">
               <string>The color of the background.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="layoutSubSearch">
         <item row="0" column="1">
          <widget class="QLabel" name="labelAuxSearch">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Auxiliary Search</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelSubList_2">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Subtitle List</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkSubListTimestamps">
           <property name="text">
            <string>Show timestamps</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkSubListWindow">
           <property name="toolTip">
            <string>Opens the subtitle list in a separate window if enabled.</string>
           </property>
           <property name="text">
            <string>Open in a new window</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkAuxSearchWindow">
           <property name="toolTip">
            <string>Opens the search in a separate window if enabled.</string>
           </property>
           <property name="text">
            <string>Open in a new window</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelMenuBar">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Menu Bar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkMenuFullscreen">
         <property name="toolTip">
          <string>Allows the menu bar to be used in fullscreen mode like in windowed mode.
This feature is not recommended because it will create a one pixel border around the entire window in fullscreen mode.
This is due to bugs between Qt, OpenGL, Windows, and some graphics card drivers.
The one pixel border does fix some bugs such as popups not displaying in fullscreen with some graphics card drivers.
Enable only if you don't mind the border.</string>
         </property>
         <property name="text">
          <string>Enable in fullscreen (Not Recommended)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetDpiScaling" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelDpiScaling">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>DPI Scaling</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutDpiScaleOverride">
            <item alignment="">
             <widget class="QCheckBox" name="checkDpiScaleOverride">
              <property name="toolTip">
               <string>When using fractional DPI scaling on Wayland, set this value to your DPI scale</string>
              </property>
              <property name="text">
               <string>Override DPI scaling factor to</string>
              </property>
             </widget>
            </item>
            <item alignment="">
             <widget class="QDoubleSpinBox" name="spinDpiScaleFactor">
              <property name="minimum">
               <double>0.050000000000000</double>
              </property>
              <property name="maximum">
               <double>99.950000000000003</double>
              </property>
              <property name="singleStep">
               <double>0.050000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelStylesheets">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Stylesheets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkStyleSheets">
         <property name="toolTip">
          <string>Style sheets allow for interface customization with CSS-like syntax
Click the help button for documentation</string>
         </property>
         <property name="text">
          <string>Enable custom style sheets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameStyleSheets">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelSubList">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Subtitle List</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="editSubList">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>600</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelSplitter">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Player-Subtitle List Splitter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="editSplitter">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>600</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelDefinitions">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Definitions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="editDefinitions">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>600</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Help|QDialogButtonBox::StandardButton::Reset|QDialogButtonBox::StandardButton::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
